test/pubsub/dummy-subscriber/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG BUILDPLATFORM="linux/amd64"
+ARG BUILDERIMAGE="golang:1.20-bullseye"
+
+FROM --platform=$BUILDPLATFORM $BUILDERIMAGE as builder
+
+ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT=""
+ARG LDFLAGS
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=${TARGETOS} \
+    GOARCH=${TARGETARCH} \
+    GOARM=${TARGETVARIANT}
+
```

Comment:
we are missing distroless base image here? see  https://github.com/open-policy-agent/gatekeeper/blob/master/test/externaldata/dummy-provider/Dockerfile

Reply:
looks like this is not resolved, i don't see usage of distroless base image